moveit_ros/warehouse/warehouse/src/initialize_demo_db.cpp

```diff
@@ -110,9 +110,8 @@ int main(int argc, char** argv)
   {
     const robot_model::JointModelGroup* jmg = psm.getRobotModel()->getJointModelGroup(gnames[i]);
     if (jmg == nullptr)
-    {
       continue;
-    }
+
```

Comment:
Even if the parentheses were removed, it was confirmed that the code had the same execution result.